docs/sources/operators-guide/deploy-grafana-mimir/getting-started-helm-charts/_index.md

```diff
@@ -38,6 +38,20 @@ Verify that you have:
 
 > **Note:** Although this is not strictly necessary, if you want to access Mimir from outside of the Kubernetes cluster, you will need an ingress. This procedure assumes you have an ingress controller set up.
 
+### Security setup
+
+This installation will not succeed if you have enabled the
+[PodSecurityPolicy admission controller](https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy)
```

Comment:
```suggestion
[PodSecurityPolicy](https://v1-23.docs.kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podsecuritypolicy) admission controller
```

Reply:
are you sure? the links are to the admission controller docs

Reply:
we don't have "admisson controller" as part of the link in the other cases, so it's for consistency only.

Reply:
it's a different link in other places

Reply:
I understand it's a different link, my comment is to make the linking consistent, either make "admission controller" part of the clickable link or not. Also maybe we should actually write "Pod Security Policy" as that's the display name, PodSecurityPolicy is the object name on the API